pub/assets/js/hound.js

```diff
@@ -284,12 +284,13 @@ var Model = {
   UrlToRepo: function(repo, path, line) {
     var info = this.repos[repo],
         url = info.url.replace(/\.git$/, ''),
-        anc = line ? '#L' + line : '';
+        anc = line ? info.anchorpattern.replace('${line}', line) : '';
```

Comment:
Won't this break if anchorpattern isn't set in the config? We need defaults on the JS side as well as the Go side I think.